crates/sui-framework-build/src/compiled_package.rs

```diff
@@ -249,6 +260,72 @@ impl CompiledPackage {
 
         Ok(())
     }
+
+    // If true, generatour layout schemas for all types declared by this package, as well as
```

Comment:
nit, typo

```suggestion
    // If true, generate our layout schemas for all types declared by this package, as well as
```

Reply:
Also, what needs to be true here to genete the layout schemas?

Reply:
Good point--this is a leftover comment from when I gated generation with a flag here instead of in the CLI--will fix